src/models/thread.ts

```diff
@@ -556,14 +556,18 @@ export class Thread extends ReadReceipt<EmittedEvents, EventHandlerMap> {
 
     /**
      * Determines if the given user has read a particular event ID with the known
-     * history of the room. This is not a definitive check as it relies only on
+     * history of the thread. This is not a definitive check as it relies only on
```

Comment:
sorry, I still don't understand this sentence. What does it mean to have read an event with the known history of the thread (or a room?)

Reply:
It can only iterate through the events that are loaded client-side, and is not going to backfill the room to find a definitive answer. It will only work against what is given to it.

Reply:
Ok, well, the doc-comment needs to say that. I've added a suggestion.